Packs/FeedGCPWhitelist/Integrations/FeedGCPWhitelist/README.md

```diff
@@ -0,0 +1,59 @@
+Use the Google Cloud Platform whitelist integration to get indicators from the feed.
```

Comment:
Update this line to be similar/identical to what you have in the `_description.md` file.